spec/integration/provider/acl/windows_spec.rb

```diff
@@ -48,10 +48,12 @@ def get_permissions_for_path(path)
   end
 
   before :each do
+    skip unless Puppet.features.microsoft_windows?
     resource.provider = provider
   end
 
   it 'throws an error for an invalid target' do
+    skip ('Not on Windows platform') unless Puppet.features.microsoft_windows?
```

Comment:
Do we need these individual skips if there's a skip in the `before :each`?

Reply:
Good catch, fixed it.